pom.xml

```diff
@@ -73,6 +73,7 @@
         <module>service/http.directory</module>
         <module>service/http.proxy</module>
         <module>service/proxy</module>
+        <module>service/turn.rest</module>
```

Comment:
This should be placed after <module>test.util</module> . Otherwise, the compilation won't work. 

The module service/turn.rest depends upone the EarlyAccessFeature which is declared in ateway.util/src/main/java/org/kaazing/gateway/util/feature/EarlyAccessFeatures.java .